features/bootstrap/FeatureContext.php

```diff
@@ -856,7 +856,14 @@ public function install_wp( $subdir = '' ) {
 			if ( $install_cache_path ) {
 				mkdir( $install_cache_path );
 				self::dir_diff_copy( $run_dir, self::$cache_dir, $install_cache_path );
-				self::run_sql( 'mysqldump --no-defaults', array( 'result-file' => "{$install_cache_path}.sql" ), true /*add_database*/ );
+
+				$support_column_statistics = exec( '/usr/bin/env mysqldump --help | grep "column-statistics"' );
```

Comment:
I think this needs a better setup:
```suggestion
				$mysqldump_binary = Utils\force_env_on_nix_systems( 'mysqldump' );
				$support_column_statistics = exec( "{$mysqldump_binary} --help | grep 'column-statistics'" );
```